exercises/queen-attack/canonical-data.json

```diff
@@ -18,39 +18,54 @@
           "description": "queen with a valid position",
           "property": "create",
           "queen": {
-            "position": "(2,2)"
+            "position": {
+              "row": 2,
+              "column": 2
+            }
           },
           "expected": 0
         },
         {
           "description": "queen must have positive rank",
```

Comment:
this is very awkward that the description (and thus the keys in the JSON file) use "row" yet the description uses "rank". We'll want to fix that up later (or now??)

Reply:
I'm willing to change it now.